_posts/2015-07-16-method-equiv.md

```diff
@@ -0,0 +1,354 @@
+---
+layout: post
+title: When are two methods alike?
+
+meta:
+  nav: blog
+  author: S11001001
+  pygments: true
+---
+
+When are two methods alike?
+===========================
+
+*This is the second of a series of articles on “Type Parameters and
+Type Members”.  If you haven’t yet, you should
+[start at the beginning]({% post_url 2015-07-13-type-members-parameters %}),
+which introduces code we refer to throughout this article without
+further ado.*
+
+[In the last part]({% post_url 2015-07-13-type-members-parameters %}),
+we just saw two method types that, though different, are effectively
+the same: those of `plengthT` and `plengthE`.  We have rules for
+deciding when an existential parameter can be lifted into a method
+type parameter—or a method type parameter lowered to an
+existential—but there are other pairs of method types I want to
+explore that are the same, or very close.  So let’s talk about how we
+determine this equivalence.
+
+A method *r* is more general than or as general as *q* if *q* may be
+implemented by only making a call to *r*, passing along the arguments.
+By more general, we mean *r* can be invoked in all the situations that
+*q* can be invoked in, and more besides.  Let us call the result of
+this test *r* <:*ₘ* *q*, where <:*ₘ* is pronounced “party duck”; if
+the test of *q* making a call to  *r* fails, then ¬(*r* <:*ₘ* *q*).
+
+If *q* <:*ₘ* *r* and *r* <:*ₘ* *q*, then the two method types are
+*equivalent*; that is, neither has more expressive power than the
+other, since each can be implemented merely by invoking the other and
+doing nothing else.  We write this as *q* ≡*ₘ* *r*.  Likewise, if
+*r* <:*ₘ* *q* and ¬(*q* <:*ₘ* *r*), that is, *q* can be written by
+calling *r*, but not vice versa, then *r* is *strictly more general*
+than *q*, or *r* <:*ₘ* *q*.
```

Comment:
The way I read this, you are saying that there are two relations, *party duck* and *strictly-party duck*. But I see the same symbol (<:*ₘ*) being used for both. Should one be <≡:*ₘ*?